README.md

```diff
@@ -243,6 +242,56 @@ In the strict mode, `async` functions are checked such that they only raise
 effects on forward declarations, callbacks and methods using
 `{.raises: [CatchableError].}` (or more strict) annotations.
 
+### Cancellation support
+
```

Comment:
This is very confusing and incorrect of what cancellation is. There is absolutely no information why cancellation is needed and where it is used, and why it can't be avoided. But there a lot of information how to swallow, bypass and ignore cancellation.

Reply:
Added more simple examples